### PR TITLE
Issue 270

### PR DESCRIPTION
**Pull request by elma13**
Small functionality added: in step 2 you can deselect station when having a look at the time series. NB the text should be added later to the json (if json is updated in English)

### CODE REVIEW

**Comment by elma13** [comment]
@Hendricxw, thsi branch can be merged after merging #269

**Review by Hendricxw — APPROVED** [review]

**Pull request merged**
